apps/dashboard/src/components/contract-components/fetch-contracts-with-versions.ts

```diff
@@ -61,21 +49,81 @@ export async function fetchPublishedContractVersions(
   return uniquePublishedContracts;
 }
 
+async function getSortedPublishedContractVersions(params: {
+  publisherAddress: string;
+  contractId: string;
+  client: ThirdwebClient;
+}) {
+  const { publisherAddress, contractId, client } = params;
+
+  const allVersions = await getPublishedContractVersions({
+    contract: getContractPublisher(client),
+    publisher: isAddress(publisherAddress)
+      ? publisherAddress
+      : await resolveAddress({
+          client,
+          name: mapThirdwebPublisher(publisherAddress),
+        }),
+    contractId: contractId,
+  });
+
+  const sortedVersions = allVersions.toSorted((a, b) => {
```

Comment:
fwiw this `toSorted` was throwing an error for me locally. I had to temporarily set it to `[...allVersions].sort()` to get it working locally. Not sure if it's an issue.